app/policies/action_policy.rb

```diff
@@ -0,0 +1,45 @@
+class ActionPolicy < ApplicationPolicy
+
+  ADMIN_OPERATIONS     = [Action::MEMO_OPERATION, Action::APPROVE_OPERATION, Action::DENY_OPERATION, Action::CANCEL_OPERATION].freeze
+  APPROVER_OPERATIONS  = [Action::MEMO_OPERATION, Action::APPROVE_OPERATION, Action::DENY_OPERATION].freeze
+  REQUESTER_OPERATIONS = [Action::CANCEL_OPERATION].freeze
+
+  class Scope < ApplicationPolicy::Scope
+    def resolve
+      raise Exceptions::NotAuthorizedError, "Read access not authorized for #{scope}" unless permission_check('read', scope)
+      return scope.all if admin?(scope.model)
+
+      action_ids = approver_id_list(scope.model.table_name)
+      Rails.logger.debug { "Approver scope for actions: #{action_ids}" }
```

Comment:
@hsong-rh This is for both user and approver correct or is this only for approver?

Reply:
So far it's only for `approver`, because `user` now doesn't have `approval:actions:read` permission yet.